samtranslator/model/sam_resources.py

```diff
@@ -1168,6 +1168,7 @@ class SamApi(SamResourceMacro):
         "Mode": PropertyType(False, IS_STR),
         "DisableExecuteApiEndpoint": PropertyType(False, is_type(bool)),
         "ApiKeySourceType": PropertyType(False, IS_STR),
+        "AlwaysDeploy": Property(False, is_type(bool)),
```

Comment:
nit and irrelavant: can we create a `IS_BOOL`

Reply:
Yes.